summary.sh

```diff
@@ -4,19 +4,19 @@
 # Use its output to guide your analysis of the results of running ./run-dljc.sh: you must manually
 # examine the results of any project that appears in the "unaccounted for" list.
 
-target=$1
+targetdir=$1
 
-total=$(find "${target}" -name "*.log" | wc -l)
+total=$(find "${targetdir}" -name "*.log" | wc -l)
```

Comment:
Why does this use find rather than ls?  Might the files be in subdirectories?

Reply:
shellcheck issued a warning on the original version, which used `ls`. I could change it back and suppress the warning, but I don't think it really matters either way.